src/TagField.php

```diff
@@ -262,6 +267,20 @@ protected function getOptions()
         $ids = $values->column($this->getTitleField());
 
         $titleField = $this->getTitleField();
+        
+        if ($this->shouldLazyLoad) {
```

Comment:
Minor: can we use `$this->getShouldLazyLoad()`?

Reply:
Done.